v2/pkg/subscraping/sources/netlas/netlas.go

```diff
@@ -0,0 +1,196 @@
+// Package netlas logic
+package netlas
+
+import (
+	"context"
+
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/projectdiscovery/subfinder/v2/pkg/subscraping"
+)
+
+type Item struct {
+	Data struct {
+		A           []string `json:"a,omitempty"`
+		Txt         []string `json:"txt,omitempty"`
+		LastUpdated string   `json:"last_updated,omitempty"`
+		Timestamp   string   `json:"@timestamp,omitempty"`
+		Ns          []string `json:"ns,omitempty"`
+		Level       int      `json:"level,omitempty"`
+		Zone        string   `json:"zone,omitempty"`
+		Domain      string   `json:"domain,omitempty"`
+		Cname       []string `json:"cname,omitempty"`
+		Mx          []string `json:"mx,omitempty"`
+	} `json:"data"`
+}
+
+type Response struct {
+	Items []Item `json:"items"`
+	Took  int    `json:"took"`
+}
+
+type DomainsCountResponse struct {
+	Count int `json:"count"`
+}
+
+// Source is the passive scraping agent
+type Source struct {
+	apiKeys   []string
+	timeTaken time.Duration
+	errors    int
+	results   int
+	skipped   bool
+}
+
+// func get_domains(offset string, domain string) int {
```

Comment:
It's a good idea to eliminate any code that's been commented out and that you're not planning on using.